### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by lokey**
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-server/317)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,223 +1,104 @@
...
 
-**Config backend**: This is the backend service that handles the sources, destinations and their connections. User management and access based roles are defined here.
+You can also set up and use RudderStack on your platform of choice. Please refer to the following guides for detailed instructions:
 
-**Customer webapp**: This is the front end application that enables the teams to set up their customer data routing with RudderStack. These will show you high-level data on event deliveries and more stats. It also provides access to custom enterprise features.
+- [Setting up RudderStack on Docker](https://docs.rudderstack.com/getting-started/installing-and-setting-up-rudderstack#setup-instructions-for-docker)
+- [Setting up RudderStack on Kubernetes](https://docs.rudderstack.com/getting-started/installing-and-setting-up-rudderstack#setup-instructions-for-kubernetes)
+- [Setting up RudderStack on a Native Installation](https://docs.rudderstack.com/getting-started/installing-and-setting-up-rudderstack#setup-instructions-for-a-native-installation)
```
Comment (lokey) [inline README.md:62]: These 3 links are redirecting to the same page (https://docs.rudderstack.com/administrators-guide/installing-and-setting-up-rudderstack). That page contains the link to the relevant docs. Can we directly link to those docs instead?

Comment (ameypv-rudder) [reply README.md:62]: Sure, I'll fix this.

File: README.md [hunk 2]
```diff
@@ -1,223 +1,104 @@
...
 
-# RudderStack Config Generator
+- **Seamless Integration**: RudderStack currently supports integration with over 30 popular destination platforms such as Google Analytics, Amplitude, Mixpanel, Amazon S3,and more. Moreover, it also supports integration with popular data warehouses such as Snowflake, Redshift, and Google BigQuery
 
-Rudderstack has two components _control plane_ and _data plane_.
-Data plane reliably delivers your event data. Control plane manages the configuration of your sources and destinations.
-This configuration can also be read from a file instead of from Control plane, if you don't want to use our hosted control plane.
+- **Client-side SDK Support**: RudderStack offers client-side SDKs for [JavaScript](https://docs.rudderstack.com/sdk-integration-guide/getting-started-with-javascript-sdk), [Android](https://docs.rudderstack.com/sdk-integration-guide/getting-started-with-android-sdk), [iOS](https://docs.rudderstack.com/sdk-integration-guide/getting-started-with-ios-sdk), and [Unity](https://docs.rudderstack.com/sdk-integration-guide/getting-started-with-unity-sdk)
```
[the remark this answers is not in the document]
Comment (ameypv-rudder) [reply README.md:23]: Maybe we can work on the docs first and then update the readme with the links later?

File: README.md [hunk 3]
```diff
@@ -1,223 +1,104 @@
...
-4. You can then login to your Google Analytics account and verify that events are delivered. Go to `MainPage->RealTime->Events`. `RealTime` view is important as the other dashboard can sometimes take 24-48 hrs to refresh.
-5. You can use our [Javascript][rudder-sdk-js-git-repo], [Android][rudder-sdk-android-git-repo] or [iOS][rudder-sdk-ios-git-repo] SDKs for sending events from your app.
+- **Extreme Scale**: One of our largest installations currently sends **300 Million** events/day with peak of **40K** req/sec, via a multi-node RudderStack setup
 
----
+- **Privacy and Security**: You can collect and store your customer data without sending everything to a third-party vendor. With RudderStack, you get fine-grained control over what data to forward to which analytical tool
 
-# RudderStack Config Generator
+- **Seamless Integration**: RudderStack currently supports integration with over 30 popular destination platforms such as Google Analytics, Amplitude, Mixpanel, Amazon S3,and more. Moreover, it also supports integration with popular data warehouses such as Snowflake, Redshift, and Google BigQuery
```
[the remark this answers is not in the document]
Comment (ameypv-rudder) [reply README.md:21]: Definitely, I'll update the number here.

File: README.md [hunk 4]
```diff
@@ -1,223 +1,104 @@
...
-9. Run the backend server `go run -mod=vendor main.go`
-10. Follow (Send Test Events) instructions below to send test event.
+RudderStack's backend is written in Go, with a rich UI written in React.js.
 
----
+You can also use the [cloud-hosted](https://app.rudderstack.com/signup?type=freetrial&utm_source=github&utm_medium=rdr-srv&utm_campaign=hosted&utm_content=intro) RudderStack instance to experience the product. Click [here](https://app.rudderstack.com/signup?type=freetrial&utm_source=github&utm_medium=rdr-srv&utm_campaign=hosted&utm_content=intro).
 
-# Send Test Events
+# Why Use RudderStack?
+- **Production-ready**: Companies like **MatterMost**, **IFTTT**, **Grofers**, and **1mg** use RudderStack for collecting their events
```
Comment (SumanthPuram) [inline README.md:15]: Mattermost.

File: README.md [hunk 5]
```diff
@@ -1,223 +1,104 @@
...
-4. You can then login to your Google Analytics account and verify that events are delivered. Go to `MainPage->RealTime->Events`. `RealTime` view is important as the other dashboard can sometimes take 24-48 hrs to refresh.
-5. You can use our [Javascript][rudder-sdk-js-git-repo], [Android][rudder-sdk-android-git-repo] or [iOS][rudder-sdk-ios-git-repo] SDKs for sending events from your app.
+- **Extreme Scale**: One of our largest installations currently sends **300 Million** events/day with peak of **40K** req/sec, via a multi-node RudderStack setup
 
----
+- **Privacy and Security**: You can collect and store your customer data without sending everything to a third-party vendor. With RudderStack, you get fine-grained control over what data to forward to which analytical tool
 
-# RudderStack Config Generator
+- **Seamless Integration**: RudderStack currently supports integration with over 45 popular destination platforms such as Google Analytics, Amplitude, Mixpanel, Amazon S3,and more. Moreover, it also supports integration with popular data warehouses such as Snowflake, Redshift, and Google BigQuery
```
Comment (SumanthPuram) [inline README.md:21]: Let's link to https://docs.rudderstack.com/destinations

Remove `such as` for warehouses. We only support 3 now.
Also link to https://docs.rudderstack.com/data-warehouse-integrations

File: README.md [hunk 6]
```diff
@@ -1,223 +1,107 @@
...
-All the source and destination configuration stays on your local storage. You can export/import config to a JSON file.
+- **User-specified Transformation**: RudderStack gives you the power to filter or transform your events before sending them to the desired destinations
 
-## Setup
+# Key Features
 
-1. `cd utils/config-gen`
-2. `npm install`
-3. `npm start`
+- **Segment API Compatible**: RudderStack is Segment API and library compatible. So you don't need to change your app if you are using Segment.
```
Comment (SumanthPuram) [inline README.md:29]: RudderStack is Segment API compatible. So you don't need to change your app if you are using Segment, just integrate RudderStack SDKs into your app.

File: README.md [hunk 7]
```diff
@@ -1,223 +1,107 @@
...
 
-![image](https://blobscdn.gitbook.com/v0/b/gitbook-28427.appspot.com/o/assets%2F-Lq586FOQtfjJPKbd01W%2F-M0LidOVklHOkLYEulS4%2F-M0M3fzyrb-UHiNBG7j0%2FScreenshot%202020-02-18%20at%2012.20.57%20PM.png?alt=media&token=a3f24ad8-fe72-4fed-8953-8e4c790f6cfd)
+- **Unlimited Events**: Event volume-based pricing of most of the commercial systems is broken. With RudderStack, you are be able to collect as much data as possible without worrying about overrunning event budgets.
 
-## Export workspace config
+- **Stand-alone System**: RudderStack runs as a single Go binary with the only dependency being on the PostgreSQL database. There is no other dependency required to use RudderStack.
 
-After adding the required sources and destinations, export your workspace config. This workspace-config is required by the RudderStack Server.
-To learn more about adding sources and destinations in RudderStack, refer [Adding a Source and Destination in RudderStack](https://docs.rudderstack.com/getting-started/adding-source-and-destination-rudderstack)
+- **High Performance**: On a single m4.2xlarge, RudderStack can process 3000 events/second.
```
Comment (SumanthPuram) [inline README.md:37]: Add 1 more section: *Platform independent* RudderStack is Kubernetes native and can be run in any Kubernetes cluster. RudderStack is cloud-agnostic and can be run on stand-alone machines in all cloud platforms.

@ameypv-rudder Add a link to the following repo for the Kubernetes setup.
https://github.com/rudderlabs/rudderstack-helm

File: README.md [hunk 8]
```diff
@@ -1,223 +1,107 @@
...
-## Start RudderStack with the workspace config file
 
-- Download the workspace config file on your machine.
-- In `docker-compose.yml`, uncomment `volumes` section under `backend` service. Specify the path to your workspace config.
-- In `build/docker.env`, set the environment variable `RSERVER_BACKEND_CONFIG_CONFIG_FROM_FILE=true`
+# RudderStack Architecture
 
----
+RudderStack has two major components, namely:
+- The Control Plane: Allows you to manage the configuration of your sources and destinations
```
Comment (SumanthPuram) [inline README.md:45]: Can you add some kind of styling to emphasize on the words `Control Plane` and `Data Plane`

Comment (SumanthPuram) [reply README.md:45]: Config-generator should be introduced here. We should talk about it before the RudderStack Setup. 

Two options for setting up the connections:
1. Managed control plane
2. Open source Config generator

**Pull request merged**